src/main/java/org/jabref/gui/fieldeditors/FieldEditors.java

```diff
@@ -25,16 +27,39 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.jabref.model.entry.FieldName.AUTHOR;
+import static org.jabref.model.entry.FieldName.INSTITUTION;
+import static org.jabref.model.entry.FieldName.TITLE;
+import static org.jabref.model.entry.FieldName.YEAR;
+
 public class FieldEditors {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(FieldEditors.class);
 
-    public static FieldEditorFX getForField(String fieldName, TaskExecutor taskExecutor, DialogService dialogService, JournalAbbreviationLoader journalAbbreviationLoader, JournalAbbreviationPreferences journalAbbreviationPreferences, JabRefPreferences preferences, BibDatabaseContext databaseContext, String entryType, SuggestionProviders suggestionProviders, UndoManager undoManager) {
+    private static final Set<String> SINGLE_LINE_FIELDS = Collections.unmodifiableSet(new HashSet<>(
```

Comment:
Can you please add this list two `InternalBibtexFields` (this class needs to be refactored at some point, but for now it is good to have anything at one place).

Reply:
Yep. Sorry for delay, I will do that on Saturday.